src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewComponentResult.cs

```diff
@@ -47,17 +37,32 @@ public class ViewComponentResult : ActionResult
         public Type ViewComponentType { get; set; }
 
         /// <summary>
+        /// Get the view data model.
+        /// </summary>
+        public object Model => ViewData?.Model;
+
+        /// <summary>
         /// Gets or sets the <see cref="ViewDataDictionary"/> for this result.
         /// </summary>
         public ViewDataDictionary ViewData { get; set; }
 
         /// <summary>
+        /// Gets or sets the <see cref="ITempDataDictionary"/> for this result.
+        /// </summary>
+        public ITempDataDictionary TempData { get; set; }
```

Comment:
what's the purpose of moving these properties? They were in alphabetical order (not required), but at least that's **an** order.














And yes, I'm making an example out of how every change that goes into a PR is a potential point of disagreement/feedback.